### PR TITLE
Calculate appliance run start

### PR DESCRIPTION
**Pull request by theis999**
- Added menu_run_appliance enum.
- Added calculate_appliance_run function.

calculate_appliance_run takes an appliance and a date,
Then calculates when it is most environmentally friendly to run that appliance within a period.
The period is determined appliance runTime * 24.

It then outputs that the best time is either now, or a number of hours later

image
_____
It currently does not handle appliances with a non-integer runtime. It is also not exactly clear that it should be handled as:
```
double runTime; // time to run appliance rounded to whole hour
```

### CODE REVIEW

File: PowerHouse/MenuFunctions.c [hunk 1]
```diff
@@ -69,3 +70,87 @@ void graph_draw(void)
...
+
+    Appliance app = Appliances[app_index];
+
+// specify date  
+    tm day;
+    day.tm_year = 2022;
+    day.tm_min = 0;
+    day.tm_sec = 0;
+    day.tm_isdst = -1;
+    printf("Which day do you wish to see the graph for?\n");
```
Comment (chrnigi) [inline PowerHouse/MenuFunctions.c]: Change prompt to reflect the current menu.

**Review by Head9x — APPROVED** [review]
Compiles and runs. No errors spotted. Approved for merging

**Review by chrnigi — APPROVED** [review]
Change the copied prompt, else good.

File: PowerHouse/MenuFunctions.c (hunk 1 above)
Comment (theis999) [reply PowerHouse/MenuFunctions.c]: Fair i missed that.

**Comment by theis999** [comment]
I discovered that the appliance was using **Wh** while the formula was `Carbon Intensity gCO2/kWh (LCA) `
So i divided the result 1000.

**Pull request merged**